### PR TITLE
reformat performance notifications

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -111,13 +94,13 @@ MATCH (c:Child), (p:Parent) RETURN c, p
 Returned GQLSTATUS code::
 03N90
 
-Returned Status Description::
+Returned StatusDescription::
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:97]: so it should be status description?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-156.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-156.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -2270,90 +2090,204 @@ Verify that this is the intended behavior of your query or command.
...
+|Descriptions
+a|
+- The user already has the role. See Status Codes documentation for more information.
+- The user does not have the role. See Status Codes documentation for more information.
+- The role already has the privilege. See Status Codes documentation for more information.
+- The role does not have the privilege. See Status Codes documentation for more information.
 |Category
 m|SECURITY
+|GQLSTATUS code
+m|00N71
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:2102]: looks like, GQL status has to be 00N70

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 3]
```diff
@@ -2270,90 +2090,204 @@ Verify that this is the intended behavior of your query or command.
...
+|Status description
+|note: successful completion - role or privilege not assigned.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:2104]:
```suggestion
|note: successful completion - role or privilege already assigned.
```

**Pull request merged**